lib/tasks/pending_imports_reminder.thor

```diff
@@ -0,0 +1,13 @@
+# -*- encoding: utf-8 -*-
+
+class PendingImportsReminder < Thor
```

Comment:
Curious about introducing thor here. So far all our lib/tasks are rake. Would be nice to maintain consistency unless there's some major reason to leverage thor here.

Reply:
Because it is much easier to create option arguments in thor. If you need to send reminders in different frequency depending on number of pending imports that users have, you would want to pass number of imports as option. For example, every week, we send reminders to all users with pending imports with `thor pending_imports_reminder:process`. And, every other day, we send reminders to users with 10 or more pending imports with `thor pending_imports_reminder:process --minimum 10`. You could pass options with rake task, but ugly like `rake 'pending_imports_reminder[10]'`.

Reply:
We don't pass an argument though right?

Reply:
Fair enough. Let's give it a try.